aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -89,6 +89,8 @@ func TestAccAWSElasticacheReplicationGroup_basic(t *testing.T) {
 						"aws_elasticache_replication_group.bar", "cluster_mode.#", "0"),
 					resource.TestCheckResourceAttr(
 						"aws_elasticache_replication_group.bar", "number_cache_clusters", "2"),
+					resource.TestCheckResourceAttrSet(
+						"aws_elasticache_replication_group.bar", "member_clusters"),
```

Comment:
Understandably since you were not able to run the acceptance testing, you weren't able to discover this test failure:

```
=== RUN   TestAccAWSElasticacheReplicationGroup_basic
--- FAIL: TestAccAWSElasticacheReplicationGroup_basic (1009.96s)
    testing.go:518: Step 0 error: Check failed: Check 4/5 error: aws_elasticache_replication_group.bar: Attribute 'member_clusters' expected to be set
```

The reason this happens is because `schema.TypeSet` attributes do not actually save information at just the attribute name. Specifically for `schema.TypeSet` attributes, the following is set:
* `NAME.#` - the number of elements present
* `NAME.HASH` - the value of an element in the set, where HASH is computed by `schema.HashString` as defined in the schema

Here is an example of the Terraform state from one of my test runs (NOTE: since the identifiers are randomized in the test, the hash values will change):

```
  member_clusters.# = 2
  member_clusters.166139442 = tf-gx1hjkvphs-002
  member_clusters.2431534984 = tf-gx1hjkvphs-001
```

So long story short, to fix the test here, we can switch it to use the following:

```go
resource.TestCheckResourceAttr("aws_elasticache_replication_group.bar", "member_clusters.#", "2"),
```

A similar change needed to be applied in the data source testing.